src/test/java/io/openliberty/tools/intellij/it/SingleModMPLSTestCommon.java

```diff
@@ -316,7 +316,7 @@ public static void prepareEnv(String projectPath, String projectName) {
 
         // pre-open project tree before attempting to open files needed by testcases
         ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
-        JTreeFixture projTree = projectFrame.getProjectViewJTree(projectName);
```

Comment:
The comment suggests that this call to `getProjectViewTree()` was intentional and Andy/Ed might have thought it was necessary to have here. Did you mean to remove this? They would more likely have a better understanding of why this was here, assuming they remember.

Reply:
I only removed it because the result was not used but it might be there for side effects. Unfortunately I wrote this some months ago so I will need to redo the investigation.

Reply:
My reason was that projTree was not used but now projectFrame is not used. Restoring old code.